### ISSUE TITLE
Sheet Cell bool values are never set in Cell Model

### ISSUE THREAD

**Issue by akrherz** [smartsheet-platform/smartsheet-python-sdk#38]
A `CHECKBOX` type column will have a JSON cell representation like so in the API

`{u'columnId': 3...4, u'value': True}`

This `value` of True never gets set in the Cell model due to

``` python
    @value.setter
    def value(self, value):
        if isinstance(value, six.string_types):
            self._value = value
```

So it is impossible to determine the value of a `CHECKBOX`, or am I missing something? :)  Thank you!

**Comment by lincolnlorscheider** [comment]
I'm pretty sure there's a better way of doing this, but I changed my version to 

`    @value.setter
    def value(self, value):
        if isinstance(value, six.string_types) or value:
            self._value = value`
which worked, for my purposes.

**Comment by timwellswa** [comment]
This was resolved with PR for 2.0.10 API changes:

```
    @value.setter
    def value(self, value):
        if isinstance(value, (six.string_types, six.integer_types, float, bool)):
            self._value = value
```
